app/src/main/res/values/strings.xml

```diff
@@ -175,27 +175,27 @@
     <string name="version_name_title">Version name:</string>
     <string name="backup_mode_title">Backup mode:</string>
     <string name="last_backup_title">Last backup:</string>
-    <string name="app_s_type_title">App\'s type:</string>
```

Comment:
Not sure what is happening here. "App type" is a valid substitute in nb_NO, etc.